### ISSUE TITLE
Cross creation fails silently if lowercased parent name is not distinct

### ISSUE THREAD

**Issue by bellerbrock** [solgenomics/sgn#1452]
Expected Behavior 
--------------------------------------------------------------------------

For Bugs:
---------

### Enviroment 

sweetpotatobase.org
#### Steps to Reproduce

Manage->Crosses->Create New Cross. Added a cross using Beauregard as a parent. Got a success message, but no cross was created, and log contained:
```
Name in pedigree is not a stock
Parent Tanzania or Beauregard in pedigree is not a stock
There were 1 invalid crosses
Invalid pedigrees in array.  No crosses will be added
```
The root cause is the search in ``` _get_stock_resultset_exact``` in Stock::Lookup, which fails if the lowercased parent name isn't distinct. In this case there was an erroneously added accession BEAUREGARD. Would be good to make this more clear in the error messages, then return those error messages rather than a success.

**Comment by titima15** [comment]
@bellerbrock I couldn't reproduce this problem. I tested this issue by changing accession names from uppercased to lowercased  and the pop up dialog informed that the stock is not in the database.

**Comment by bellerbrock** [comment]
fixed in d686ff7bdd561bde6ff6b34126030c3917bbfc03
